src/Microsoft.NetCore.Analyzers/Core/Performance/DoNotUseCountWhenAnyCanBeUsed.cs

```diff
@@ -236,14 +235,14 @@ private static void OnCompilationStart(CompilationStartAnalysisContext context)
         private sealed class OperationActionsHandler
         {
             private readonly INamedTypeSymbol targetType;
```

Comment:
Please use `_` as the prefix for instance field names. I am planning to bump up the naming violations to warning level to aid here.

Reply:
This can be done in your other open PR. I will merge this one.